docs/sdkx-react-native/release-notes-react-native.mdx

```diff
@@ -23,6 +23,14 @@ import {
 
 ## 10.2.3
 
+_June 2nd, 2023_
+
+** Release Notes: **
+
+- Added an API to pause/resume the In-app notifications. For more details, refer our [Notifications](/sdkx-react-native/notifications-ios/#pause-in-app) page.
```

Comment:
We can just mention like this:
For more details, refer: [Notifications](/sdkx-react-native/notifications-ios/#pause-in-app).

Reply:
@sumeet-helpshift  updated.